web/js/components/CardsContainer.js

```diff
@@ -0,0 +1,25 @@
+import Card from './Card.js'
+
+export default async function CardsContainer() {
+  const response = await fetch(
+  'https://api.github.com/repos/vikiival/trap/contents/images?ref=master'
```

Comment:
Why does it fetch it from the repo when the images are stored locally?

Reply:
@vikiival these files are just statically hosted, and right now, I am not aware of any way we can list all available images in the folder 🤔